helpers/auth_helper.py

```diff
@@ -0,0 +1,29 @@
+"""An auth_helper (Singleton) is created in the module, which will be used
+    in the application to authenticate users.
```

Comment:
The first line is indented with three symbols left to it, but the second with fourth ones. This does not look beautiful.

Reply:
Removed indentation.